src/components/Search.js

```diff
@@ -1,15 +1,34 @@
 import React from 'react'
 
-const Search = () => {
-  return (
-    <div className="ui huge fluid icon input">
+export default class Search extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      searchTerm: ""
+    }
+  }
+
+  onInputChange(term){
+    this.setState({
+      searchTerm: term
+    }, () => this.props.handleChange(this.state.searchTerm))
```

Comment:
yeah, this is repetitive. You don't need to store the search term twice. Remember single source of truth for your data. This component can just call the function on props to handle it.